src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.eligibility.unit.spec.js

```diff
@@ -179,9 +179,11 @@ describe('VAOS Page: VAFacilityPage eligibility check', () => {
       await waitFor(() => {
         screen.queryByText(/San Diego VA Medical Center/i);
       });
-      expect(screen.baseElement).to.contain.text(
-        'You’ll need to call to schedule at this facility',
-      );
+      expect(
```

Comment:
I noticed in yesterday's PRs that this unit test is flaky on the CI and fails locally on my machine. Updating it to await `findByText` ensured the test passes reliably on my local machine so I'm making this small fix along with the other changes in this PR.